tensorflow/python/compiler/tensorrt/trt_convert.py

```diff
@@ -775,6 +777,15 @@ def get_rewriter_config(self):
     return get_tensorrt_rewriter_config(
         conversion_params=self._conversion_params)
 
+  def set_calib_tables(self, calib_tables):
+    """Set calibration tables into converted graph definition."""
+    assert self._converted, "Calibration tables can only be set on converted graph"
```

Comment:
How to use this feature? Does it accept a TF-TRT converted savedmodel, convert it again (which should be a noop), and then set the calibration table? If that is the case, please make a separate tool for it instead of reusing TrtGraphConverter. Otherwise, please comment and add a test.

Reply:
@aaroey I have updated the PR description with the use case where this feature helps users (1) skip passing large calibration dataset (2) skip lengthy calibration step. Calibration tables can be loaded or saved from converter only when user is converting FP32 model to INT8.